### PR TITLE
[Metricbeat] Add Oracle overview dashboard

### PR DESCRIPTION
**Pull request by sayden**
Adds an overview dashboard for Oracle. I also took advantage to fix a couple of typos I found

~https://user-images.githubusercontent.com/4249331/66646611-fda6f300-ec26-11e9-9469-8ba4ab7fa4b9.png~
~https://user-images.githubusercontent.com/4249331/66646633-0d263c00-ec27-11e9-9022-d2b9b9dc545d.png~

image

### CODE REVIEW

File: CHANGELOG.next.asciidoc [hunk 1]
```diff
@@ -423,6 +423,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Leverage KUBECONFIG when creating k8s client. {pull}13916[13916]
 - Add ability to filter by tags for cloudwatch metricset. {pull}13758[13758] {issue}13145[13145]
```
Comment (kaiyan-sheng) [inline CHANGELOG.next.asciidoc]: why we have an extra line here? 😬

File: metricbeat/docs/fields.asciidoc [hunk 2]
```diff
@@ -23746,7 +23746,7 @@ type: keyword
 *`oracle.performance.io_reloads`*::
 +
 --
-Reloads / Pins ratio. A Reload is any PIN of an object that is not the first PIN performed since the object handle was created, and which requires loading the object from disk. Pins are tumber of times a PIN was requested for objects of this namespace
+Reloads / Pins ratio. A Reload is any PIN of an object that is not the first PIN performed since the object handle was created, and which requires loading the object from disk. Pins are the number of times a PIN was requested for objects of this namespace
```
Comment (kaiyan-sheng) [inline metricbeat/docs/fields.asciidoc]: missing period `.` after namespace.

Comment (sayden) [reply metricbeat/docs/fields.asciidoc]: Thanks Kaiyan! Fixed!

**Comment by exekias** [comment]
Thank you for getting this in! Will you open a backport to 7.5?

**Pull request merged**